service/discovery/azure/azure.go

```diff
@@ -162,7 +162,8 @@ type clients struct {
 	networkSecurityGroupsClient *armnetwork.SecurityGroupsClient
 
 	// AppService
-	sitesClient *armappservice.WebAppsClient
+	sitesClient           *armappservice.WebAppsClient
+	appServiceFarmsClient *armappservice.PlansClient
```

Comment:
Why is it called XFarmsClient and not PlansClient?

Reply:
For me, it sounded more intuitive. But yes, the new name is `plans`. I will change it

Reply:
Funny enough, sometimes in the properties they still use "farms", e.g. "sites.Properties.ServerFarmID"